packages/gestalt/src/Video/Playhead.js

```diff
@@ -30,9 +30,14 @@ export default class VideoPlayhead extends PureComponent<Props, State> {
   seek: (clientX: number) => void = (clientX) => {
     if (this.playhead) {
       const { duration, seek } = this.props;
-      const { left, width } = this.playhead.getBoundingClientRect();
-      const percent = Math.max(0, Math.min((clientX - left) / width, 1));
+      const { left, right, width } = this.playhead.getBoundingClientRect();
+
+      const isRTL = window.getComputedStyle(this.playhead).direction === 'rtl';
```

Comment:
Did some googling and seems a fine solution https://stackoverflow.com/questions/35170581/how-to-access-styles-from-react

Reply:
@AlbertCarreras yes, we can omit `window` but I wanted to make it more explicit that the method is a global one.

Reply:
I think perf wise `getComputedStyle()` has a cost to calculate everytime we seek. 

Since we have a pattern of `rtl` on the HTML tag, could we just read this from the html root element attribue without computing the style

something like:
```
const isRtl = document.getElement('html')?.getAttribute("dir") === "rtl"
```